responsibleai/tests/test_model_analysis_validations.py

```diff
@@ -35,6 +35,25 @@ def test_validate_unsupported_task_type(self):
                 target_column='target',
                 task_type='regre')
 
+    def test_validate_test_data_size(self):
+        X_train, X_test, y_train, y_test, _, _ = \
+            create_iris_data()
+
+        model = create_lightgbm_classifier(X_train, y_train)
+        X_train['target'] = y_train
+        X_test['target'] = y_test
+
+        with pytest.raises(UserConfigValidationException) as ucve:
+            ModelAnalysis(
```

Comment:
Can you check the message in the exception as well here?

Reply:
I do below.